module-types/Cisco/DS-C48-300AC.yml

```diff
@@ -0,0 +1,12 @@
+---
+manufacturer: Cisco
+model: DS-C48-300AC
+part_number: DS-C48-300AC
+weight: 2.42
+weight_unit: lb
+comments: 'Cisco MDS 9148 Multilayer Fabric Switch'
+power-ports:
+  - name: Input
```

Comment:
Probably best to name this `PSU{module}` so that port name conflicts don't occur